pilot/pkg/model/telemetry.go

```diff
@@ -126,10 +126,10 @@ type metricsConfig struct {
 
 type telemetryFilterConfig struct {
 	metricsConfig
-	Provider      *meshconfig.MeshConfig_ExtensionProvider
-	Metrics       bool
-	AccessLogging bool
-	LogsFilter    *tpb.AccessLogging_Filter
+	Provider              *meshconfig.MeshConfig_ExtensionProvider
+	Metrics               bool
+	AccessLogging         bool
+	StackdriverLogsFilter *tpb.AccessLogging_Filter
```

Comment:
why is this renamed to `StackdriverLogsFilter`?  The logging filter should be applicable for all logs. Does this PR disable support for basic envoy logs filtering?

Reply:
no, basic envoy logs are using `AccessLogging`, right now only SD need `telemetryFilters`.